README.md

```diff
@@ -97,326 +90,75 @@ Then, replace the *oidc* settings in `.samples.config.json` to point to your new
+More information on how to configure and use this middleware library can be found [here](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware#expressoidc-api)
 
-1. Choose the front-end
+In this sample, the [okta oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware) library is configured to use the values in `.samples.config.json`. When the `login()` function is called from the view, the middleware library calls the [`/authorize`](http://developer.okta.com/docs/api/resources/oauth2.html#authentication-request) endpoint to start the [Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1).
 
-    | Framework | NPM module | Github |
-    |-----------|------------|--------|
-    | Angular 1 | [@okta/samples-js-angular-1](https://www.npmjs.com/package/@okta/samples-js-angular-1) | https://github.com/okta/samples-js-angular-1 |
-    | React | [@okta/samples-js-react](https://www.npmjs.com/package/@okta/samples-js-react) | https://github.com/okta/samples-js-react |
-    | Elm | [@okta/samples-elm](https://www.npmjs.com/package/@okta/samples-elm) | https://github.com/okta/samples-elm |
+You can read more about the `ExpressOIDC` configuration options here: [OpenID Connect with Okta oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware#new-expressoidcconfig).
 
-2. Install the front-end
-
-    ```bash
-    # Use the NPM module for the front-end you want to install. I.e. for React:
-    [samples-nodejs-express-4]$ npm install @okta/samples-js-react
-    ```
-
-3. Restart the server. You should be up and running with the new front-end!
+**Important:** When the authorization code is exchanged for an `access_token` and/or `id_token`, the tokens **must** be [validated](#validation). The middleware library handles this validation on the client's behalf.
 
 ## Back-end
-To complete the [Authorization Code Flow](https://tools.ietf.org/html/rfc6749#section-1.3.1), your back-end server performs the following tasks:
-  - Handle the [Authorization Code code exchange](https://tools.ietf.org/html/rfc6749#section-1.3.1) callback
-  - [Validate](http://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation) the `id_token`
-  - Set `user` session in the app
+To display the user information and handle session close, your back-end server performs the following tasks:
+  - Handle the callback from [okta oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware) after authentication
+  - Render the user profile information returned by the middleware library
   - Log the user out
 
+> Note: As you can see, the back-end has delegated handling the authorization code exchange, token validation, getting the userinfo etc to [okta oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware).
```

Comment:
This statement is implied by the  section above. We can probably remove this.

Reply:
I think it's ok to have additional information about this. We don't explicitly mention this anywhere. To a novice developer, this might be helpful.